lib/Model.js

```diff
@@ -122,7 +122,7 @@ export default class Model extends ReplicateObject {
     return prediction;
   }
 
-  async createPrediction(input) {
+  async createPrediction(input, { webhook, webhookEventsFilter } = {}) {
```

Comment:
Why does it take two objects? Couldn't this just be one object that looks something like {input = {}, webhook, webhookEventsFilter} ?

(Dupe of my comment here: https://github.com/replicate/replicate-js/pull/20#discussion_r1128706554)

Reply:
Going to deal with this in #20.